packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/CustomUserControlViewManagerCPP.cpp

```diff
@@ -115,20 +121,21 @@ void CustomUserControlViewManagerCpp::DispatchCommand(
 }
 
 // IViewManagerWithExportedEventTypeConstants
-IMapView<hstring, IInspectable> CustomUserControlViewManagerCpp::ExportedCustomBubblingEventTypeConstants() noexcept {
-  auto constants = winrt::single_threaded_map<hstring, IInspectable>();
-  return constants.GetView();
+ConstantProvider CustomUserControlViewManagerCpp::ExportedCustomBubblingEventTypeConstants() noexcept {
+  return nullptr;
 }
 
-IMapView<hstring, IInspectable> CustomUserControlViewManagerCpp::ExportedCustomDirectEventTypeConstants() noexcept {
-  auto constants = winrt::single_threaded_map<hstring, IInspectable>();
+ConstantProvider CustomUserControlViewManagerCpp::ExportedCustomDirectEventTypeConstants() noexcept {
+  return [](winrt::Microsoft::ReactNative::IJSValueWriter const &constantWriter) {
+    constantWriter.WriteObjectBegin();
 
-  auto registration = winrt::single_threaded_map<hstring, IInspectable>();
-  registration.Insert(L"registrationName", box_value(L"onLabelChanged"));
+    constantWriter.WritePropertyName(L"topLabelChanged");
```

Comment:
>WritePropertyName [](start = 19, length = 17)

While this low level API usage works, it would be better if all of this would be part of some internal code, while developers use strongly typed code and simple structs/methods. In that case it would be similar to native modules.